sdk/src/assertions/bmff_hash.rs

```diff
@@ -255,13 +270,16 @@ impl AssertionCbor for BmffHash {}
 
 impl AssertionBase for BmffHash {
     const LABEL: &'static str = Self::LABEL;
-    const VERSION: Option<usize> = Some(ASSERTION_CREATION_VERSION);
+    const VERSION: Option<usize> = Some(ASSERTION_CREATION_VERSION); // todo: this mechanism needs to change since a struct could support different versions
```

Comment:
Would you ever be generating different versions of the assertion in the same claim?

Reply:
No we always generate the latest supported.